report-hazard/index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <script src="report-hazard.js" defer></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://unpkg.com/maplibre-gl@2.4.0/dist/maplibre-gl.js"></script>
+	<link href="https://unpkg.com/maplibre-gl@2.4.0/dist/maplibre-gl.css" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css" />
+    <!-- <script type="module" src="script.js"></script> -->
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <h1>CampBuddy</h1>
+    </header>
+    <div class="toolbar">
+        <!-- header area shared among all pages. It also include navigation buttons -->
+        <a title="Select the location" href="#step1">1</a>
+        <span>-----</span>
+        <a title="What type of hazard would you like to report?" href="#step2">2</a>
+        <span>-----</span>
+        <a title="What type of wild animal did you spot?" href="#step3">3</a>
+        <span>-----</span>
+        <a title="Would you like to mention any further details?" href="#step4">4</a>
+        <span>-----</span>
+        <a title="Please upload photos to help others see the reported hazard visually." href="#step5">5</a>
+        <span>-----</span>
+        <a title="Review the report before submitting." href="#step6">6</a>
+        <span>-----</span>
+        <a href="#step7">7</a>
+    </div>
+    <main>
+        <form id="form">
+            <div class="page" id="step1">
+                <h1>1. Select the location.</h1>
+                <h1 id="loc"></h1>
```

Comment:
there cannot be 2 h1 in the same section, do we need this  empty h1?

Reply:
commit 24bcefd63ba6716b93ea31aeeac082e486ca67be

Reply:
commit 938efb1f732b3ffa9e5441b3c271ef9d3f0f0223